### DISCUSSION TITLE
Not Compatible with V2

### DISCUSSION THREAD

**Discussion by Popugames** [WalletConnect/WalletConnectUnity#65] [Q&A]
xPortal wallet stoped v1 connections. Our dapps made with Unity3D are down.

**Comment by Yozaak** [comment]
We are facing similar issues within our game. Swift resolution by team would be most appreciated.

**Comment by sael-you** [comment]
UP !

**Reply by sael-you** [reply]
Got some news from ChainOfIndustry :
https://github.com/chainofindustry/MultiversXUnityTools/issues/7#issuecomment-1458260841

**Comment by Dilosen** [comment]
Did you resolve?
